.prettierignore

```diff
@@ -3,3 +3,4 @@ translations
 scripts/js/lib/api/testdata
 .mypy_cache
 public/images
+README.md
```

Comment:
Prettier wanted the MDX example in the README to have a trailing comma, which wouldn't be valid JSON.

````diff
--- a/README.md
+++ b/README.md
@@ -600,7 +600,7 @@ We also have a specialized admonition for Qiskit Code Assistant prompt suggestio
   prompts={[
     "# Print the version of Qiskit we're using",
     "# Return True if the version of Qiskit is 1.0 or greater",
-    "# Install Qiskit 1.0.2"
+    "# Install Qiskit 1.0.2",
   ]}
 />
 ```
````

I've just ignored this; it doesn't seem to be a problem with other MDX files, and I don't think we need to lint the README.

Reply:
Sg. Formatting the README has caught contributors in the past in an annoying way